ODLA/platforms/magicmind/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+# ==============================================================================
+# Copyright (C) 2020-2021 Alibaba Group Holding Limited.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+# ==============================================================================
+
+set(NEUWARE_HOME /usr/local/neuware/)
+set(DEFAULT_COMPILER ${CMAKE_CXX_COMPILER})
+find_library(MAGICMIND_LIB NAMES magicmind PATHS ${NEUWARE_HOME} PATH_SUFFIXES lib64 NO_DEFAULT_PATH)
+if(NOT MAGICMIND_LIB)
+  message(STATUS "not find MagicMind")
```

Comment:
message(FATAL_ERROR "..." ) if lib is not found.
or  we can do "find_library(MAGICMIND_LIB ... REQUIRED)" in line 19 and remove lines 20 -25